dbms/tests/queries/0_stateless/00933_test_fix_extra_seek_on_compressed_cache.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CURDIR/../shell_config.sh
+
+
+$CLICKHOUSE_CLIENT --query="DROP TABLE IF EXISTS test.small_table"
+
+$CLICKHOUSE_CLIENT --query="CREATE TABLE test.small_table (a UInt64 default 0, n UInt64) ENGINE = MergeTree() PARTITION BY tuple() ORDER BY (a);"
+
+$CLICKHOUSE_CLIENT --query="INSERT INTO test.small_table(n) SELECT * from system.numbers limit 100000;"
+
+cached_query="SELECT count() FROM test.small_table where n > 0;"
+
+$CLICKHOUSE_CLIENT --use_uncompressed_cache=1 --query="$cached_query" &> /dev/null
+
+$CLICKHOUSE_CLIENT --use_uncompressed_cache=1 --query_id="test-query-uncompressed-cache" --query="$cached_query" &> /dev/null
+
+$CLICKHOUSE_CLIENT --query="SYSTEM FLUSH LOGS"
+sleep 1
```

Comment:
This should be before `SYSTEM FLUSH LOGS` so that SystemLog background thread has time to pull log items from queue to data vector.